src/main/java/dev/ikm/tinkar/sandbox/TinkarStarterData.java

```diff
@@ -2685,15 +2755,67 @@ private static void configureConceptsAndPatterns(StarterData starterData, UUIDUt
                 .tinkarBaseModelMembership()
                 .build();
 
+        session.compose((ConceptAssembler conceptAssembler) -> conceptAssembler.concept(TinkarTerm.LOGIC_GRAPH_FOR_SEMANTIC))
+                .attach((FullyQualifiedName fqn) -> fqn
+                        .text("Logic graph for semantic (SOLOR)")
+                        .language(ENGLISH_LANGUAGE)
+                        .caseSignificance(DESCRIPTION_NOT_CASE_SENSITIVE)
+                        .attach(new USDialect()
+                                .acceptability(PREFERRED))
+                        .attach((Synonym synonym) -> synonym
+                                .text("Logic graph")
+                                .caseSignificance(DESCRIPTION_NOT_CASE_SENSITIVE)
+                                .language(ENGLISH_LANGUAGE)))
+                .attach(new USDialect()
+                        .acceptability(PREFERRED))
+                .attach((Definition definition) -> definition
+                        .text("Semantic")
+                        .caseSignificance(DESCRIPTION_NOT_CASE_SENSITIVE)
+                        .language(ENGLISH_LANGUAGE))
+                .attach((Identifier identifier) -> identifier
+                        .source(TinkarTerm.UNIVERSALLY_UNIQUE_IDENTIFIER)
+                        .identifier(TinkarTerm.LOGIC_GRAPH_FOR_SEMANTIC.asUuidArray()[0].toString()))
+                .attach(new StatedNavigation()
+                        .parents(TinkarTerm.SEMANTIC_PROPERTIES))
+                .attach(new StatedAxiom()
+                        .isA(TinkarTerm.SEMANTIC_PROPERTIES));
+
+
         starterData.concept(TinkarTerm.LONG)
```

Comment:
You'll want to remove the code for concepts built from the old "starterData.concept" method once you migrate them to composer, otherwise it'll get added twice.